src/Raven.Server/ServerWide/Commands/ModifyDatabaseWatchersCommand.cs

```diff
@@ -64,14 +65,22 @@ public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
             if (Watcher == null)
                 return null;
 
-            var newTaskKey = Watcher.GetTaskKey().ToString();
-            if (Watcher.CurrentTaskId != null && Watcher.CurrentTaskId != newTaskKey)
+            var newTaskId = (Watcher.Database + Watcher.Url).ToLower();
```

Comment:
OTOH, that might be nice https://github.com/ravendb/ravendb/pull/2555/files#diff-df05c10a5583ee61df7c65497a1d187dR15